src/style.css

```diff
@@ -1 +1,90 @@
-/* add styles here */
+body {
+  font-family: ' Avenir ', sans-serif;
```

Comment:
The font-family declaration should include a fallback font. It's a good practice to have a generic font family specified in case the primary font fails to load. For example: `font-family: 'Avenir', Arial, sans-serif;`.